modules/tensor_mechanics/test/tests/3D_slipweakening/tpv2053D.i

```diff
@@ -0,0 +1,340 @@
+    [restore_y]
+      type = TagVectorAux
+      vector_tag = 'restore_tag'
+      v = 'disp_y'
+      variable = 'resid_y'
+      execute_on = 'TIMESTEP_END'
+    []
+    [restore_z]
+      type = TagVectorAux
+      vector_tag = 'restore_tag'
+      v = 'disp_z'
+      variable = 'resid_z'
+      execute_on = 'TIMESTEP_END'
+    []
+    [StaticFricCoeff]
+      type = FunctionAux
+      variable = mu_s
+      function = func_static_friction_coeff_mus
+      execute_on = 'LINEAR TIMESTEP_BEGIN'
+    []
+    [StrikeShearStress]
+      type = FunctionAux
+      variable = ini_shear_stress
+      function = func_initial_strike_shear_stress
+      execute_on = 'LINEAR TIMESTEP_BEGIN'
+    []
+  []
+
+  [Kernels]
+    [./inertia_x]
+      type = InertialForce
+      use_displaced_mesh = false
+      variable = disp_x
+    []
+    [./inertia_y]
+      type = InertialForce
+      use_displaced_mesh = false
+      variable = disp_y
+    []
+    [./inertia_z]
+      type = InertialForce
+      use_displaced_mesh = false
+      variable = disp_z
+    []
+    [./Reactionx]
+      type = StiffPropDamping
+      variable = 'disp_x'
+      component = '0'
+    []
+    [./Reactiony]
+      type = StiffPropDamping
+      variable = 'disp_y'
+      component = '1'
+    []
+    [./Reactionz]
+      type = StiffPropDamping
+      variable = 'disp_z'
+      component = '2'
+    []
+  []
+
+  [Materials]
+    [elasticity]
+        type = ComputeIsotropicElasticityTensor
+        lambda = 32.04e9
+        shear_modulus = 32.04e9
+        use_displaced_mesh = false
+    []
+    [stress]
+        type = ComputeLinearElasticStress
+    []
+    [density]
+        type = GenericConstantMaterial
+        prop_names = density
+        prop_values = 2670
+    []
+    [./czm_mat]
+        type = SlipWeakeningFriction3d
+        disp_slipweakening_x     = disp_slipweakening_x
+        disp_slipweakening_y     = disp_slipweakening_y
+        disp_slipweakening_z     = disp_slipweakening_z
+        reaction_slipweakening_x = resid_slipweakening_x
+        reaction_slipweakening_y = resid_slipweakening_y
+        reaction_slipweakening_z = resid_slipweakening_z
+        mu_s = mu_s
+        ini_shear_sts = ini_shear_stress
+        boundary = 'Block0_Block1'
+    [../]
+  []
+
+  [Functions]
+    [func_static_friction_coeff_mus]
+      type = PiecewiseConstant
+      axis=x
+      x = '-1000e3 -15e3 15e3'
+      y = '10000 0.677 10000.0'
+      direction = left
+    []
+    [func_initial_strike_shear_stress]
+      type = PiecewiseConstant
+      axis=x
+      x = '-1000e3 -9.0e3 -6.0e3 -1.5e3  1.5e3  6.0e3  9.0e3'
+      y = ' 70.0e6 78.0e6 70.0e6 81.6e6 70.0e6 62.0e6 70.0e6'
+    []
+  []
+
+  [UserObjects]
+    [recompute_residual_tag]
+        type = ResidualEvaluationUserObject
+        vector_tag = 'restore_tag'
+        force_preaux = true
+        execute_on = 'TIMESTEP_END'
+    []
+  []
+
+  [Executioner]
+    type = Transient
+    dt = 0.005
+    num_steps = 100
```

Comment:
It only makes it to step 40 in debug before it times out.  Can you change the number of steps and the mesh size so that it will run faster.  Like 20 steps?